### ISSUE TITLE
Cleanup JSON library usage

### ISSUE THREAD

**Issue by lukeeey** [DragonetMC/DragonProxy#473]
We currently have both Jackson and GSON which do pretty much the same thing and they are both used throughout the code.

In the future, we should use only one of these libraries.

**Comment by sgdc3** [comment]
+1 for jackson as it can be used for both configs and data mapping

**Comment by lukeeey** [comment]
Yep. Annoying thing is that there is no (obvious) support for setters that modify the value in the config file.
